client/components/social-buttons/google.js

```diff
@@ -41,7 +41,10 @@ class GoogleLoginButton extends Component {
 		error: '',
 		showError: false,
 		errorRef: null,
-		isDisabled: true,
+		isDisabled: false,
+		isLoading: false,
```

Comment:
`isLoading` and `isInitialized` are very similar names, and it's not immediately clear what each of them represents, so the distinction should probably be made clearer if they're both to remain.

Reading carefully through the code, however, it appears that `isLoading` is a special state that only gets used when the user clicks the button before the library has been initialized, and gets displayed until either the library loads or an error happens? If so, that condition can be derived from the rest of state and doesn't need to be its own entry. Something like `this.state.isClicked && ! this.state.initialized && ! this.state.error`, perhaps?